### PR TITLE
Pull feature flags every hour for LLM and LLD

### PR DESCRIPTION
**Pull request by KVNLS**
### ✅ Checklist

- [ ] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

As we are at 0.01% of usage with 12 hours we can reduce the interval to pull FF.

### ❓ Context

- **JIRA or GitHub link**: https://ledgerhq.atlassian.net/browse/LIVE-13552

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

File: apps/ledger-live-desktop/src/renderer/components/FirebaseRemoteConfig.tsx [hunk 1]
```diff
@@ -135,7 +135,7 @@ export const FirebaseRemoteConfigProvider = ({
     };
     fetchAndActivateConfig();
     // 12 hours fetch interval. TODO: make this configurable
```
Comment (LucasWerey) [inline apps/ledger-live-desktop/src/renderer/components/FirebaseRemoteConfig.tsx:137]:
```suggestion
    // 1 hour fetch interval. TODO: make this configurable
```

File: apps/ledger-live-mobile/src/components/FirebaseRemoteConfig.tsx [hunk 2]
```diff
@@ -39,7 +39,7 @@ export const FirebaseRemoteConfigProvider = ({
     };
     fetchConfig();
     // 12 hours fetch interval. TODO: make this configurable
```
Comment (LucasWerey) [inline apps/ledger-live-mobile/src/components/FirebaseRemoteConfig.tsx:41]:
```suggestion
    // 1 hour fetch interval. TODO: make this configurable
```

**Review by mcayuelas-ledger — COMMENTED** [review]
I think we can use FF to make it ? 

Or use` libs/ledger-live-common/src/config/sharedConfig.ts` to share same value one time :)

**Comment by KVNLS** [comment]
> I think we can use FF to make it ?
> 
> Or use` libs/ledger-live-common/src/config/sharedConfig.ts` to share same value one time :)

I agree, the idea here is to quickly unlock our refresh rate. But I will create a ticket to do it when we have time with a  remote configuration.

**Review by LucasWerey — APPROVED** [review]

**Review by cgrellard-ledger — APPROVED** [review]

**Pull request merged**